janus_core/calculations/md.py

```diff
@@ -4,6 +4,7 @@
 from functools import partial
 from itertools import combinations_with_replacement
 from math import isclose
+from os.path import getmtime
```

Comment:
do we need os.path? cannot be done via Pathlib?

Reply:
indeed https://docs.python.org/3/library/pathlib.html#pathlib.Path.stat this can do it

Reply:
I did look into that, but don't think we can directly use it as a `sorted` `key` in the same way since you access it through the `st_mtime` attribute, rather than a function.

I think we could still use it through a lambda function, but it seemed unnecessarily messy.

Given that `Path.stat` returns an `os.stat_result` object anyway, I'm not sure we would gain much anyway?

Reply:
`operator.attrgetter` would do it but it wouldn't necessarily be cleaner.

Reply:
Thanks. Unless someone definitely thinks it's a cleaner alternative I think I'll leave it as it is.

Reply:
ok